SUMMARY.md

```diff
@@ -0,0 +1,28 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > Ternary operator is a shorthand way of doing if/else statements. It holds three operands: a condition, a result for true, and a result for false. You will learn how you can use the ternary operator in PHP.
+
+* [Type Hinting](content/TypeHinting/TypeHinting.q.md) ** - minutes**
+    > Discover how you can use type hinting to specify the expected data type in a function declaration.
+
+* [Cookies](content/Cookies/Cookies.q.md) ** - minutes**
+    > Learn how to create/retrieve cookies, modify cookie values, delete cookies, and check if cookies are enabled.
+ 
+* [Sessions](content/Sessions/Sessions.q.md) ** - minutes**
+    > Learn how to start a PHP session, get PHP session variable values, modify a PHP session variable, and destroy a PHP session.
+
+* [Serialization](content/Serialization/Serialization.q.md) ** - minutes**
+    > Serialization in PHP is used for storing and passing values while retaining their type and structure. In this lesson, you will learn how you can use `serialize()` and `unserialize()` methods.
+
+* [Error Handling](content/ErrorHandling/ErrorHandling.q.md) ** - minutes**
+    > Error handling refers to the process of catching errors detected in the program that requires proper action. Find out how you can handle errors using different types of error handling, error logs, and error handling functions.
+    
+* [Catching Exceptions](content/CatchingExceptions/CatchingExceptions.q.md) ** - minutes**
+    > Catching an exception is the new object-oriented way of dealing with errors through exception handling. It searches for a `catch` statement that can handle the type of exception. You will learn how you can handle errors with `try-catch` statements and basic exception classes.
```

Comment:
Rephrase this `You will learn how you can handle errors with `try-catch` statements and basic exception classes.`
to:
> In this lesson, you will learn how to handle errors using `try-catch` statements and basic exception classes.

OR
> In this lesson, you will learn how to use the basic PHP `Exception` classes and create the `try` and `catch` statements in handling errors.